docs/hoon/twig/bar-core/hep-loop.md

```diff
@@ -12,10 +12,12 @@ sort: 4
 
 ```
 :rap  $
-:core
-++  $  p
---
-``` 
+:trap  p
+```
+
+```
+=<($ |.(p))
+```
 
```

Comment:
I replaced the explicit core with trap, because of the rune description: "form a trap and kick it".